scripts/ci/generate-changelog

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Looks fragile to me. Have you looked into https://github.com/conventional-changelog/conventional-changelog with a custom preset?

Reply:
I agree this looks fragile. But let me explain why we went with this approach.

Within our dream team we discussed using conventional-changelog to generate changelogs. There are a few issues:
- it generates the changelog in a format not the same as the dcos/dcos PR template. So it would require us to write another (fragile) scripting phase to convert whatever it generates to the [template format](https://github.com/dcos/dcos/blob/master/PULL_REQUEST_TEMPLATE.md).
- conventional-changelog causes more changes for how we do releases. It would increment the version in `package.json` (which we currently *never* change). Also I have lots of questions regarding how this would all play with EE, since it doesn't have a `package.json`.

Me and @bstavroulakis think we should should this as-is, then make a follow-up ticket to improve the changelog in the future.

Reply:
@ahoskins sounds good, can you link the follow-up ticket here?